server/lib/routes/get-index.js

```diff
@@ -10,8 +12,12 @@ module.exports = function (config) {
   route.path = '/';
 
   route.process = function (req, res) {
+
+    var flowEventData = flowMetrics(config, req);
+
     res.render('index', {
-      flowBeginTime: Date.now(),
+      flowBeginTime: flowEventData.flowBeginTime,
+      flowId: flowEventData.flowId,
```

Comment:
The test `response.render was called correctly` in `tests/server/routes/get-index.js` needs updating to cover `flowId`. Assuming that tests are coming for the new server `flow-metrics` module, you can just mock it out and assert against the mocked value.